files/grolemund-2016-09.md

```diff
@@ -0,0 +1,65 @@
+
+*   Course material is open source, which lets many instructors work together to refine it.
+
+*   Lessons have a useful infrastructure:
+    *   Workshop webpage
+    *   Online course notes and exercises
+￼
+## Negative
+
+*   Includes some material that is not useful, and does not bring material together in a way that can be quickly applied in a classroom.
+
+*   Provides an incomplete picture of what it takes to build a mental model because it does not connect working memory to cognitive load theory or mention a main extension of Cognitive Load Theory (Multimedia Learning Theory):
+    *   Learning tasks involve three types of cognitive load: inherent, generative, and external
+    *   What's missing: a person only has a finite amount of cognitive load to expend at any given moment. Inherent load exceeds this bandwidth, hence you need to scaffold the presentation of complex material. The student must choose to expend generative load, hence you need to show the student how to expend generative load in an efficient way and then motivate them to expend it. Extraneous load diminishes this bandwidth, hence you need to reduce distractions.
+
+*   Multimedia learning theory (not covered)
+    *   Available load is distributed across visual and verbal information channels.
+    *   Material is easier to learn when split simultaneously across different formats.
+    *   It is more difficult to learn when multiple types of information are presented simultaneously in the same format.
+    *   This presents a clear case for visual slides and insights on how to avoid bad slides.
+
+*   Due to the above, SC does not use slides when they would be appropriate, which makes their own material difficult to learn (especially in the instructor workshop).
+    *   Along with the above, SC does not discuss how to make effective slides, when to use them, and how to interleave them with live coding
+
+*   Does not discuss how to keep energy levels and student participation high in a workshop.
+
+*   Does not provide a clear view of what peer-led instruction would look like in a workshop.
+
+*   Training does not cover solutions to common logistical problems: installation issues, multiple operating systems, ensuring prerequisite knowledge, disseminating materials, etc.
+
+*   Spends a lot of time on live teaching exercises that are too artificial to be helpful.
+    *   Teachers do not get to use projectors, slides, or computers (to code) during the exercises, which makes them de facto communication style evaluations.
+    *   This is useful, but justifies less time.
+
+*   Introduces the idea of concrete learning objectives, but does not explain how learning objectives relate to creating a mental model.
+
+*   Does not clearly connect the content of the instructor training to the content of the Software Carpentry workshops.
+    *   As a result, students do not understand how to apply the educational psychology in the classroom.
```

Comment:
That's certainly the feedback I get pretty consistently at IT. MCQs are one of the best examples. We used to have some but the problem is they were not related with the sample dataset (for example, for command line). So instead of the participants being able to try out ("What happens if you do: sort *.dat | grep "76" >> sorted.txt....") they were presented with a question that indeed was a good MCQ but they had to do it in their heads or on the paper. Lots of people were checking out.

We could combine our exercises with some MCQs, for example.